cms/envs/common.py

```diff
@@ -780,7 +780,7 @@
     'DEFAULT_FROM_EMAIL', 'BULK_EMAIL_DEFAULT_FROM_EMAIL',
     'FEATURES', 'MKTG_URLS', 'MARKETING_SITE_ROOT',
     'CORS_ORIGIN_WHITELIST', 'CURRENT_PLAN', 'LMS_SEGMENT_KEY', 'CMS_SEGMENT_KEY', 'BADGR_USERNAME', 'BADGR_PASSWORD',
-    'BADGR_ISSUER_SLUG', 'ENABLE_OPENBADGES',
+    'BADGR_ISSUER_SLUG', 'ENABLE_OPENBADGES', 'BADGR_FLAG'
```

Comment:
kindly remove `ENABLE_OPENBADGES` now as we have new config

Reply:
Done